public/api-docs/v1/change_requests.yaml

```diff
@@ -0,0 +1,103 @@
+"/api/v1/planning_applications/{planning_application_id}/change_requests":
+  get:
+    summary: Retrieves all determined planning applications
+    security:
+      - bearerAuth: []
+    parameters:
+      - in: path
+        name: planning_application_id
+        required: true
+        schema:
+          type: integer
+          minimum: 1
+        description: The planning application ID
+      - in: query
+        name: change_access_id
+        schema:
+          type: string
+        required: true
+        description: A unique access code for this planning application
+
+    responses:
+      "200":
+        description: All change requests
+        content:
+          application/json:
+            examples:
+              Full:
+                value:
+                  data:
+                    - id: 1
+                      type: description_change_request
+                      proposed_description: A proposed new description
+                      state: open
+                      response_due: "2020-05-14T05:18:17.540Z"
+                      approved: true
+                      rejection_reason: nil
+
+
+"/api/v1/planning_applications/{planning_application_id}/description_change_requests":
```

Comment:
I removed the ID because of this comment you made about it already being part of the params? https://github.com/unboxed/bops/pull/270#discussion_r620313488